src/scripts/runscripts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+COUNTER=0 # The count of tests run
+FAIL=0 # The count of failed tests
+SUCCESS=0 # The count of successful tests
+for i in $(ls $1); do
```

Comment:
Please add a check for arg count, and a `--help` options to describe the usage of the script.